x-pack/plugins/ml/public/application/management/jobs_list/components/jobs_list_page/jobs_list_page.tsx

```diff
@@ -37,123 +35,19 @@ import { PLUGIN_ID } from '../../../../../../common/constants/app';
 
 import { checkGetManagementMlJobsResolver } from '../../../../capabilities/check_capabilities';
 
-// @ts-ignore undeclared module
-import { JobsListView } from '../../../../jobs/jobs_list/components/jobs_list_view';
-import { DataFrameAnalyticsList } from '../../../../data_frame_analytics/pages/analytics_management/components/analytics_list';
-import {
-  ModelsList,
-  getDefaultModelsListState,
-} from '../../../../trained_models/models_management/models_list';
 import { AccessDeniedPage } from '../access_denied_page';
 import { InsufficientLicensePage } from '../insufficient_license_page';
 import { JobSpacesSyncFlyout } from '../../../../components/job_spaces_sync';
-import { getDefaultAnomalyDetectionJobsListState } from '../../../../jobs/jobs_list/jobs';
 import { getMlGlobalServices } from '../../../../app';
-import { ListingPageUrlState } from '../../../../../../common/types/common';
-import { getDefaultDFAListState } from '../../../../data_frame_analytics/pages/analytics_management/page';
 import { ExportJobsFlyout, ImportJobsFlyout } from '../../../../components/import_export_jobs';
-import type { JobType, MlSavedObjectType } from '../../../../../../common/types/saved_objects';
-import { useMlKibana } from '../../../../contexts/kibana';
-
-interface Tab extends EuiTabbedContentTab {
-  'data-test-subj': string;
-}
-
-function usePageState<T extends ListingPageUrlState>(
-  defaultState: T
-): [T, (update: Partial<T>) => void] {
-  const [pageState, setPageState] = useState<T>(defaultState);
-
-  const updateState = useCallback(
-    (update: Partial<T>) => {
-      setPageState({
-        ...pageState,
-        ...update,
-      });
-    },
-    [pageState]
-  );
+import type { MlSavedObjectType } from '../../../../../../common/types/saved_objects';
```

Comment:
Not changed by this PR, but there is no link from model IDs in the management page trained model list back to the model in the ML app. How hard would it be to add in this PR? Currently there is no quick way to jump back to the ML models list, as there is for Anomaly detection and DFA jobs.